torch/csrc/cuda/Stream.cpp

```diff
@@ -60,12 +60,41 @@ static PyObject * THPVariable_get_cuda_stream(THCPStream *self) {
   END_HANDLE_TH_ERRORS
 }
 
+static PyObject * THPVariable_get_priority(THCPStream *self) {
+  HANDLE_TH_ERRORS
+  return PyLong_FromLong(self->cuda_stream.priority());
+  END_HANDLE_TH_ERRORS
+}
+
+static PyObject * THCPStream_priority_range(THCPStream *self) {
+  HANDLE_TH_ERRORS
+  THPObjectPtr tuple(PyTuple_New(2));
+  if (!tuple) return nullptr;
+
+  int least_priority, greatest_priority;
+  std::tie(least_priority, greatest_priority) =
+    self->cuda_stream.priority_range();
+
+  PyTuple_SET_ITEM(tuple.get(), 0, PyLong_FromLong(least_priority));
+  PyTuple_SET_ITEM(tuple.get(), 1, PyLong_FromLong(greatest_priority));
```

Comment:
PyLong_FromLong can also potentially fail. Instead of checking `PyLong_FromLong` and `PyTuple_New` call, you can combine everything with one `Py_BuildValue` which will simplify the code:

```
return Py_BuildValue("(ii)", least_priority, greatest_priority);
```

`()` means tuple
`i` means int

https://docs.python.org/3/c-api/arg.html?highlight=buildvalue#c.Py_BuildValue